### PR TITLE
fix(1010): [2][small] replace prChain flag with chainPR annotation.

### PR DESCRIPTION
**Pull request by wdstar**
## Context

We moves prChain flag from the top level setting `prChain` to the pipeline level annotation `chainPR`.

## Note

This PR persists the annotation value to the **existing** `pipeline.prChain` property, therefore it does not impact the others (e.g. UI). 
https://github.com/screwdriver-cd/models/pull/344/files#diff-679e90b5b15ab1b73ddb7ed2377e7413R637

## References

- https://github.com/screwdriver-cd/screwdriver/issues/1010#issuecomment-466188824
- https://github.com/screwdriver-cd/screwdriver/pull/1532

### CODE REVIEW

**Review by yuichi10 — APPROVED** [review]
LGTM

**Review by tk3fftk — APPROVED** [review]
LGTM:+1:

File: test/lib/pipeline.test.js [hunk 1]
```diff
@@ -527,23 +527,24 @@ describe('Pipeline Model', () => {
...
-                    'beta.screwdriver.cd/executor': 'screwdriver-executor-vm'
+                    'beta.screwdriver.cd/executor': 'screwdriver-executor-vm',
+                    'screwdriver.cd/chainPR': true
                 });
             });
         });
 
         it('stores prChain to pipeline', () => {
             const configMock = Object.assign({}, PARSED_YAML);
+            const defatulChainPR = false;
```
Comment (tkyi) [inline test/lib/pipeline.test.js:538]: small type
```suggestion
            const defaultChainPR = false;
```

**Review by tkyi — APPROVED** [review]

**Comment by wdstar** [comment]
typo 😢

**Pull request merged**